### ISSUE TITLE
[Package issue] react-transition-group

### ISSUE THREAD

**Issue by drwpow** [pikapkg/cdn#56]
Carried over from #37, importing the following URL `https://cdn.skypack.dev/-/react-transition-group@v4.4.1-JClfB1cJgoafz5Apsp4Q/dist=es2020,mode=raw/esm/utils/PropTypes.js` yields the following error:

```
process is not defined
```

This is a raw import from `react-bootstrap`, which we do a little transformation on. But it seems `process` isn’t being properly handled.

**Comment by soegaard** [comment]
Following.

**Comment by soegaard** [comment]
Is there anything I can do to help resolve this error?

I have found the blog post [1] in which I learned that `process.env.NODE_ENV` is used to distinguish 
between production and development. Since it variable `process` is a Node thing, build tools such 
as WebPack and Browserify does a code transform to remove conditionals involving the variable.

Since Skypack is intended to work without a build step, I am unsure, whether I as user can do anything.
For example: Is it possible to inject a fake `process.env` into the JavaScript environment in order to
prevent the "not defined" error? Maybe by loading a custom "Skypack Fake Env" module first?

[1] https://hub.packtpub.com/building-better-bundles-why-processenvnodeenv-matters-optimized-builds/
